### PR TITLE
Feature: Support for Import of Charts of accounts

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts [hunk 1]
```diff
@@ -30,9 +30,43 @@ export class GeneralConfigurationComponent implements OnInit {
   showPaymentsField: boolean;
   showAutoCreate: boolean;
   showJeSingleCreditLine: boolean;
+  isChartOfAccountsEnabled: boolean;
+  allAccountTypes: [];
```
Comment (ashwin1111) [inline src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:34]: type type type pliss
`allAccountTypes: string[];`

File: src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts [hunk 2]
```diff
@@ -30,9 +30,43 @@ export class GeneralConfigurationComponent implements OnInit {
...
 
   constructor(private formBuilder: FormBuilder, private qbo: QboComponent, private billsService: BillsService, private settingsService: SettingsService, private route: ActivatedRoute, private router: Router, private snackBar: MatSnackBar, public dialog: MatDialog) { }
 
+  getAccountType(region) {
```
Comment (ashwin1111) [inline src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:38]: `region: string`

Comment (ashwin1111) [reply src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:38]: if you can default a value, if else is not needed

Comment (Shwetabhk) [reply src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:38]: yes do something like
common = [x, y, z]
 if us:
  common.append[a,b,c]
 if uk:
   ...

Comment (NileshPant1999) [reply src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:38]: okay cool

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**